src/backend/launcher.ts

```diff
@@ -983,11 +1013,25 @@ function getRunnerCallWithoutCredentials(
   const modifiedCommand = [...command]
   // Redact sensitive arguments (Authorization Code for Legendary, token for GOGDL)
```

Comment:
could make this a separate method called `redactSensitiveArguments`

Reply:
yeah, I think it is something like this in heroic right now, I will do that in another PR though, there are other changes I want to get for this launcher file from there as well.